k8s/autopilot/autopilot.go

```diff
@@ -0,0 +1,33 @@
+package autopilot
+
+import (
+	autopilotclientset "github.com/libopenstorage/autopilot-api/pkg/client/clientset/versioned"
+	"k8s.io/client-go/rest"
+)
+
+// AutopilotRuleOps provides an interface to Autopilot operations.
```

Comment:
I think it would be good to have a singleton instance of individual clients just like we have it for the k8sOps object. That way the users can just invoke the Instance anywhere without creating a new instance every time. 
```
import autopilotOps "github.com/portworx/sched-ops/k8s/autopilot"

autopilotOps.Instance().GetAutopilotRule(...)
```
We can do this lazily just like we did it before. In that case the individual implementations will look like this - 
```
func (c *Client) CreateAutopilotRule(rule *autv1alpaha1.AutopilotRule) (*autv1alpaha1.AutopilotRule, error) {
        if err := initAutopilotClient(); err != nil {
               return nil, err
        }
	return c.autopilot.AutopilotV1alpha1().AutopilotRules().Create(rule)
}
```
Refer `initK8sClient` method for above change.

Reply:
+1

Reply:
Updated.